hybridgroup/commander
# 50 current status button

Add Activity Log, to log each command execution status.
Add a status indicator about the last command executed.
Show activity log using the status indicator.
Clear activity log when switching command sets.
## How to test
1. Run the app.
2. If no command sets load one.
3. Once on the command set page, no indicator should be shown.
4. After clicking any of the commands, the indicator should be visible, either green or red depending on the status of the command execution.
5. Click the indicator to show the activity log.
6. Load a different command set, the log and indicator should be cleared.
